viam_cartographer_test.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	testExecutableName = "true" // the program "true", not the boolean value
-	testDataFreqHz     = "5"
-	testDataRateMsec   = 200
+	testExecutableName  = "true" // the program "true", not the boolean value
+	testLidarDataFreqHz = "5"
+	testIMUDataFreqHz   = "20"
```

Comment:
[q] We currently support both for backwards compatiablity?

Reply:
Both what?

Reply:
Msec and Hz

Reply:
Nick wanted to leave all backend code using Msec, which actually makes sense since the only applications we have of the variable use msec, such that only user-facing code uses hz

Reply:
Spoke offline, will refactor